pytorch_lightning/loggers/base.py

```diff
@@ -235,7 +232,7 @@ def log_graph(self, model: "pl.LightningModule", input_array=None) -> None:
 
     def save(self) -> None:
         """Save log data."""
-        self._finalize_agg_metrics()
```

Comment:
Same comment about this method

Reply:
it is ok to remove this, because the call into `self._finalize_agg_metrics()` will ignore the branch

`if metrics_to_log is not None:`

because `metrics to log` is always None.